src/assets/scripts/client/AppController.js

```diff
@@ -265,6 +269,8 @@ export default class AppController {
             return;
         }
 
+        EventTracker.trackEvent(TRACKABLE_EVENT.AIRPORTS, 'airport-switcher', nextAirportJson.icao);
+
```

Comment:
Correct me if I am wrong but I think there should be no blank line here.

Reply:
you are correct, good catch!